Source/DOM classes/SVG-DOM/SVGRect.h

```diff
@@ -27,7 +27,7 @@ typedef struct
  
  Unfortunately, the SVG Spec authors defined "uninitialized" and "values of zero" to mean differnet things, so we MUST preserve
  that difference! */
-SVGRect SVGRectUninitialized();
+SVGRect SVGRectUninitialized(void);
```

Comment:
Why is this necessary?

Reply:
“This function declaration is not a prototype” warning